modules/nw-sriov-network-attachment.adoc

```diff
@@ -28,20 +28,35 @@ apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetwork
 metadata:
   name: <name> <1>
-  namespace: sriov-network-operator <2>
+  namespace: openshift-sriov-network-operator <2>
 spec:
   networkNamespace: <target_namespace> <3>
   ipam: |- <4>
     ...
   vlan: <vlan> <5>
   resourceName: <sriov_resource_name> <6>
+  linkState: <link_state> <7>
+  maxTxRate: <max_tx_rate> <8>
+  minTxRate: <mix_rx_rate> <9>
+  vlanQoS: <vlan_qos> <10>
+  spoofChk: <spoof_check> <11>
+  trust: <trust_vf> <12>
+  capabilities: <capabilities> <13>
```

Comment:
@pliurh - Is this another block scalar, like the following?

```
capabilities: |-
  {
    ...
  }
```

Thanks!

Reply:
@zshi Could you give some comments?

Reply:
Please see the example in https://github.com/zshi-redhat/openshift-docs/commit/73c3637472dd360aa7b5e1fa170f4229e6700555

Reply:
Yes, here is another example using block scalar: https://github.com/redhat-nfvpe/openshift-baremetal-ci/blob/master/tests/sriov/templates/sn-static-ipam.yaml
this template has been tested in CI env.